rascal-lsp/src/main/rascal/util/LanguageServer.rsc

```diff
@@ -65,14 +66,29 @@ Language language(PathConfig pcfg, str name, str extension, str mainModule, str
 alias Parser           = Tree (str /*input*/, loc /*origin*/);
 
 @synopsis{Function profile for summarizer contributions to a language server}
+@description{
+Summarizers provide information about the declarations and uses in the current file
+which can be used to populate the information needed to implement interactive IDE
+features.
+
+There are two places a Summarizer can be called:
+* Summarizers can be called after _file save_, in this case we use ((builder))s. Builders typically also have side-effects on disk (leaving generated code or API descriptions in the target folder), and they may run whole-program analysis and compilation steps.
```

Comment:
Add text that documents that `builder` expect __at least__ the same error messages as the analyzer. (or is it not that hard, it will work fine if not true?)

Reply:
It will work fine if not true; the diff will just always be empty

Reply:
okay, thats good news.